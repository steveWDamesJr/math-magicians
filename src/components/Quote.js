import React from 'react';

function Quote() {
  return (
    <main>
      <div className="flex justify-center">
        <p className="quote1">
          Mathematics is not about numbers,        equations, computations,
          or algorithms: it is about       understanding. - William Paul   Thurston
        </p>
      </div>
    </main>
  );
}

export default Quote;
