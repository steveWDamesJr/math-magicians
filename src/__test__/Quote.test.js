import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

it('Renders correctly', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});