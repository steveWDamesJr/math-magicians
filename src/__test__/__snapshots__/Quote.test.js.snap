// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<main>
  <div
    className="flex justify-center"
  >
    <p
      className="quote1"
    >
      Mathematics is not about numbers,        equations, computations, or algorithms: it is about       understanding. - William Paul   Thurston
    </p>
  </div>
</main>
`;
